chain/indexer/src/streamer/utils.rs

```diff
@@ -19,8 +19,9 @@ pub(crate) async fn convert_transactions_sir_into_local_receipts(
     let prev_block = fetch_block(&client, block.header.prev_hash).await?;
     let prev_block_gas_price = prev_block.header.gas_price;
 
-    let runtime_config =
-        node_runtime::config::RuntimeConfig::from(protocol_config.runtime_config.clone());
+    let runtime_config_store =
+        near_primitives::runtime::config_store::RuntimeConfigStore::new(None);
```

Comment:
I will likely move the creation of the `runtime_config_store` into the parent function if this ends up the right approach.